### PR TITLE
feat: account Email verification

### PR DESCRIPTION
**Pull request by evgeniy-ac**

### CODE REVIEW

File: src/dotnet/UI.Blazor/Components/Phone/EmailVerifier.razor [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+            <p class="c-description">Please confirm your email</p>
+        </div>
+    }
+</div>
+
+@code {
+    private TotpVerifier? _totpVerifier;
+    private string? _emailUnderVerification;
+
+    [Inject] private Session Session { get; init; } = null!;
```
Comment (crui3er) [inline src/dotnet/UI.Blazor/Components/Phone/EmailVerifier.razor:25]: You can inject single UIHub service.

File: src/dotnet/Users.UI.Blazor/Components/OwnAccountEditorModal.razor [hunk 2]
```diff
@@ -102,7 +149,7 @@
         _account = await Accounts.GetOwn(Session, default);
         _form.Name = _account.Name;
         _form.LastName = _account.LastName;
-        _form.Email = _account.Email;
+        _form.Email = _account.Email.IsNullOrEmpty() ? null : _account.Email;
```
Comment (crui3er) [inline src/dotnet/Users.UI.Blazor/Components/OwnAccountEditorModal.razor]: There is another extension usefull in this case: _account.Email.NullIfEmpty()

**Review by crui3er — APPROVED** [review]

**Review by alexis-kochetov — APPROVED** [review]

File: src/dotnet/UI.Blazor/Components/Phone/EmailVerifier.razor (hunk 1 above)
Comment (evgeniy-ac) [reply src/dotnet/UI.Blazor/Components/Phone/EmailVerifier.razor:25]: done

File: src/dotnet/Users.UI.Blazor/Components/OwnAccountEditorModal.razor (hunk 2 above)
Comment (evgeniy-ac) [reply src/dotnet/Users.UI.Blazor/Components/OwnAccountEditorModal.razor]: done

**Pull request merged**